src/main/sensors/barometer.c

```diff
@@ -140,6 +148,10 @@ void pgResetFn_barometerConfig(barometerConfig_t *barometerConfig)
     barometerConfig->baro_spi_csn = IO_TAG_NONE;
 #endif
 
+#ifndef BARO_EOC_PIN
+#define BARO_EOC_PIN NONE
+#endif
+
```

Comment:
Does this not go into `common_post.h`?

Reply:
I see @blckmn doing similar refactoring

Reply:
Okay, I was just wondering. In this case it makes sense to me.